torchtitan/metrics.py

```diff
@@ -127,7 +127,7 @@ def build_metric_logger(
     """
     dump_dir = job_config.job.dump_folder
     aim_config = job_config.metrics
-    save_aim_folder = aim_config.save_aim_folder
+    save_aim_folder = os.path.join(job_config.job.dump_folder, aim_config.save_aim_folder)
```

Comment:
I think the aim folder should be separate from the `dump_folder`. What is the proposed workflow for this method? Is the idea that all runs should then be copied to the central aim repo afterwards?

Reply:
in my mind dump_folder is something like `/nfs/h100/chem` and save_aim_folder `all_aim`, so that all experiments end up there. afterword the main runs are selected and moved to some other aim repo which will need to define.

Reply:
OK, sounds good.